### PR TITLE
EAPI Shenanigans

### PR DESCRIPTION
**Pull request by flappyports**

### CODE REVIEW

**Comment by flappyports** [comment]
@SoapGentoo please review

File: x11-plugins/pidgin-mbpurple/pidgin-mbpurple-0.3.0-r2.ebuild [hunk 1]
```diff
@@ -21,16 +22,16 @@ DEPEND="${RDEPEND}
 	virtual/pkgconfig"
 S=${WORKDIR}/${MY_P}
 
-pkg_setup() {
+src_prepare() {
+	default
+
 	tc-export CC
```
Comment (SoapGentoo) [inline x11-plugins/pidgin-mbpurple/pidgin-mbpurple-0.3.0-r2.ebuild]: I would do this separately in `src_configure`

File: x11-terms/xvt/xvt-2.1-r3.ebuild [hunk 2]
```diff
@@ -21,27 +21,30 @@ DEPEND="${RDEPEND}
...
-	epatch "${FILESDIR}/${PN}-int-main.patch"
+	"${FILESDIR}/${PN}-int-main.patch"
 
 	# fix segfault (bug #363883)
-	epatch "${FILESDIR}/${PN}-pts.patch"
+	"${FILESDIR}/${PN}-pts.patch"
+)
 
+src_prepare() {
+	default
```
Comment (SoapGentoo) [inline x11-terms/xvt/xvt-2.1-r3.ebuild]: remove, and change to `src_configure`

File: x11-terms/xvt/xvt-2.1-r3.ebuild [hunk 3]
```diff
@@ -21,27 +21,30 @@ DEPEND="${RDEPEND}
...
 	tc-export CC
 }
 
 src_install() {
-	dobin xvt || die "dobin failed"
+	dobin xvt
 	doman xvt.1
-	dodoc README
+	einstalldocs
```
Comment (SoapGentoo) [inline x11-terms/xvt/xvt-2.1-r3.ebuild]: :+1:

**Pull request closed without merging**